modules/orchestrator/proc-install-components-for-orchestrator-plugin.adoc

```diff
@@ -0,0 +1,79 @@
+
+* Manual installation
+* {product-very-short} helper script
+* {product-very-short} Orchestrator Infra Helm Chart
++
+These components enable the creation of the `SonataFlowPlatform` custom resource and several supporting `NetworkPolicy` resources.
+
+.Manual installation
+
+Use manual installation for production environments. Choose this method when you have installed the required components versions, such as when other applications are using OpenShift Serverless.
+
+For more information on preparing the required infrastructure, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_serverless/1.36[Red Hat OpenShift Serverless] to do the following:
+
+* Prepare for OpenShift Serverless installation
+
+* Install the OpenShift Serverless Operator
+
+* Install Knative Serving
+
+* Install Knative Eventing
+
+* Install the OpenShift Serverless Logic Operator
+
+.{product-very-short} helper script
+
+You can install the OpenShift Serverless infrastructure for the Orchestrator plugin using the {product-very-short} helper script. You can use the script in empty clusters, however, use with caution in production clusters because the script installs Operators with automatic upgrades and default settings, which can introduce unplanned changes, version conflicts, or policy misalignments.
+You can install the OpenShift Serverless infrastructure for the Orchestrator plugin in empty clusters by using the {product-very-short} helper script.
+
+[IMPORTANT]
+====
+This method of installation is not suitable for production clusters. The {product-very-short} helper script installs Operators with automatic upgrades and default settings, which can introduce unplanned changes, version conflicts, or policy misalignments.
+====
+
+For more information on controlling the installation of the Operators, see link:https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/[Install your Operator with OLM].
+
+.Procedure
+. Download the `plugin-infra.sh` script as shown in the following example:
++
+[code,terminal]
+----
+curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-${PRODUCT_VERSION}/config/profile/rhdh/plugin-infra/plugin-infra.sh # Specify the {product} version in the URL or use main
+----
+. Run the script as shown in the following example:
++
+[source,shell]
+----
+$ ./plugin-infra.sh
+----
+
+.{product-very-short} Orchestrator Infra Helm Chart
+You can use {product-very-short} Orchestrator Infra Helm Chart to install components for the Orchestrator plugins in empty clusters.
+
+[IMPORTANT]
+====
+{product-very-short} Orchestrator Infra Helm Chart is not a suitable installation method for production clusters. The method installs Operators with automatic upgrades and default settings, which might introduce unplanned changes, version conflicts, or policy misalignments.
```

Comment:
The Orchestrator Infra Helm Chart **is** suitable for production clusters. This is in fact what is used as prerequisite when you deploy the Orchestrator flavor using the Helm Chart.